extensions/interactions/InteractiveMap/directives/InteractiveMap.js

```diff
@@ -38,13 +41,17 @@ oppia.directive('oppiaInteractiveInteractiveMap', [
             HtmlEscaperService.escapedJsonToObj($attrs.longitudeWithValue)];
           $scope.zoom = (
             HtmlEscaperService.escapedJsonToObj($attrs.zoomWithValue));
+          $scope.interactionIsActive = ($scope.getLastAnswer() === null);
+          $scope.mapMarkers = [];
+
+          $scope.$on(EVENT_NEW_CARD_AVAILABLE, function(evt, data) {
```

Comment:
I think you can just do `function()` rather than `function(evt, data)`, since you don't make use of either of the args.

Reply:
Done, also changed for the other 6 interactions for which I had added before.